tools/image-audit/crawler/util/crawlerutil.js

```diff
@@ -51,6 +55,32 @@ class CrawlerUtil {
     return urls.filter((url) => this.isUrlValid(url));
   }
 
+  /**
+     * Removes JavaScript-related attributes from an element.
+     * @param {HTMLElement} element - The element to clean.
+     * @private
+     */
+  static #removeJsAttributes(element) {
```

Comment:
Like to understand why this optimization is necessary. I agree JS related attributes has no value in this context, but I'm also thinking if `data*` attributes also fall into same category.